PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,3 @@
-<!--
-Thanks for your contribution!
-
-Please take a quick look at those things down there. They're quite important.
-Really! We wrote them for you. Yes you! With utmost care. Read them.
--->
-
```

Comment:
why this deletion? is that specified in the issue?

Reply:
No. But after i changed cobot-->corobo this automatically got added.

Reply:
that's not possible

Reply:
maybe your branch is behind coala master

Reply:
sync your branch with coala master